modules/ROOT/pages/syntax/parsing.adoc

```diff
@@ -5,37 +5,57 @@
 
 [abstract]
 --
-This section provides a general overview of how Cypher parses an input String, including whitespace.
+This section provides a general overview of how Cypher parses an input String.
 --
 
-Cypher considers input Strings as a sequence of Unicode UTF-16 code points. Unicodes can generally be escaped as `\uxxx`.
+The Cypher parser takes an arbitrary input String.
+While the syntax of  Cypher is described in the following chapters,
+this chapter formulates the general rules on which characters are considered valid
+as valid input.
+
+== Using unicodes in Cypher
+Unicodes can generally be escaped as `\uxxx`.
 
 Additional documentation on escaping rules for string literals, names and regular expressions can be found here:
 
 * xref::syntax/expressions.adoc#cypher-expressions-string-literals[String literal escape sequences]
 * xref::syntax/naming.adoc#symbolic-names-escaping-rules[Using special characters in names]
 * xref::clauses/where.adoc#escaping-in-regular-expressions[Regular epxressions]
 
+The following example escapes the unicode character `A` (`\u0041`) in the keyword `MATCH`:
+```
+    M\u0041TCH (m) RETURN m;
+```
+
```

Comment:
I don't think this is a good example because it's invalid syntax?

Reply:
I ran this parser test in Cypher and it works:
`  s"M\\u0041TCH (m) RETURN m"`

Do you have a better idea?

Reply:
My bad, thought `u0041` was a space